src/plugin.js

```diff
@@ -52,8 +52,28 @@ const launchServer = () => {
   process.on('exit', stopServer);
 };
 
+const extensions = ['.css'];
```

Comment:
This was previously inside the plugin function which would [eventually allow defining other extensions via plugin options](https://github.com/wbyoung/babel-plugin-transform-postcss/issues/12#issuecomment-291519081).

I see the reason to move it was to abstract logic into `getStylesFromStylesheet`… that could move back into the scope of the plugin function if we want to support those options in the future.